Caboodle/Clipboard/Clipboard.android.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Android.Content;
+using Android.Runtime;
+using Android.Views;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Clipboard
+    {
+        public static void SetText(string text)
+        {
+            SetText(text, "text");
+        }
+
+        public static void SetText(string text, string label)
```

Comment:
I think we should remove the overloads with the `label` parameter as it provides no value. For android, we should just use `"Text"`.